apps/whale-api/src/module.api/consortium.service.ts

```diff
@@ -0,0 +1,102 @@
+import { Injectable } from '@nestjs/common'
+import { JsonRpcClient } from '@defichain/jellyfish-api-jsonrpc'
+import { DeFiDCache, TokenInfoWithId } from './cache/defid.cache'
+import { AssetBreakdownInfo, MemberDetail, MemberWithTokenInfo } from '@defichain/whale-api-client/dist/api/consortium'
+import BigNumber from 'bignumber.js'
+
+@Injectable()
+export class ConsortiumService {
+  constructor (
+    protected readonly rpcClient: JsonRpcClient,
+    private readonly defidCache: DeFiDCache
+  ) {}
+
+  private updateBurnMintAmounts (assetBreakdownInfo: AssetBreakdownInfo[], tokens: TokenInfoWithId[], key: string, value: string): void {
+    const tokenId = key.split('/')[4]
+    const memberId = key.split('/')[5]
+    const type = key.split('/')[6] === 'burnt' ? 'burned' : 'minted'
+    const token = tokens.find(t => t.id === tokenId)
+    if (token === undefined) {
+      return
+    }
+    const val = new BigNumber(value).toFixed(+token.decimal.toString())
+
+    const existingABI = assetBreakdownInfo.find(abi => abi.tokenSymbol === token.symbol)
+    if (existingABI === undefined) {
+      return
+    }
+
+    const existingMember = existingABI.memberInfo.find(mi => mi.id === memberId && mi.tokenId === token.id)
+    if (existingMember === undefined) {
+      return
+    }
+
+    existingMember[type] = val
+  }
+
+  private pushToAssetBreakdownInfo (assetBreakdownInfo: AssetBreakdownInfo[], memberId: string, memberDetail: MemberDetail, tokenId: string, tokens: TokenInfoWithId[]): void {
+    const backingAddresses: string[] = memberDetail.backingId.length > 0 ? memberDetail.backingId.split(',').map(a => a.trim()) : []
+
+    const member: MemberWithTokenInfo = {
+      id: memberId,
+      name: memberDetail.name,
+      minted: '0.00000000',
+      burned: '0.00000000',
+      backingAddresses,
+      tokenId
+    }
+
+    const token = tokens.find(t => t.id === tokenId)
+    if (token === undefined) {
+      return
+    }
+
+    const existingABI = assetBreakdownInfo.find(abi => abi.tokenSymbol === token.symbol)
+    if (existingABI === undefined) {
+      assetBreakdownInfo.push({
```

Comment:
From the design, both need`tokenDisplaySymbol` and `tokenSymbol` 
<img width="444" alt="image" src="https://user-images.githubusercontent.com/13292662/203886164-62cb59e2-4641-4ecc-a1d0-8da8dbb9d3da.png">
<img width="444" alt="image" src="https://user-images.githubusercontent.com/13292662/203886239-875d20ed-469f-440b-b25f-91da631e6337.png">